README.md

```diff
@@ -91,112 +128,153 @@ To compile this plugin, you need:
 
-* `core_energy`
+* `CORE/E`
+    or old syntax: `core_energy`
 
     Collect power consumption information for every core.
 
-* `core_power`
+* `CORE/P`
+    or old syntax: `core_power`
 
     Collect power consumption information for every core.
 
-* `gpu_energy`
+* `GPU/E`
+    or old syntax: `gpu_energy`
 
     Collect power consumption information for every gpu.
 
-* `gpu_power`
+* `GPU/P`
+    or old syntax: `gpu_power`
 
     Collect power consumption information for every gpu.
 
-* `dram_energy`
+* `DRAM/E`
+    or old syntax: `dram_energy`
 
     Collect power consumption information for every dram.
 
-* `dram_power`
+* `DRAM/P`
+   or old syntax: `dram_power`
 
     Collect power consumption information for every dram.
 
-E.g.:
+* `PACKAGE0/E`, etc.
 
-    export SCOREP_METRIC_X86ENERGY_PLUGIN="gpu_power"
+    Collect power consumption information for package on node 0.
+    This also works with every other metric.
 
-or for VampirTrace:
+E.g.:
 
-    export VT_PLUGIN_CNTR_METRICS=X86ENERGY_*_energy
+    export SCOREP_METRIC_X86_ENERGY_PLUGIN="GPU/P" or
+    export SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN="GPU/P"
 
 ### Environment variables
 
-* `SCOREP_X86ENERGY_INTERVAL_US`/`VT_X86ENERGY_INTERVAL_US` (default=100000)
+* `SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN`/`SCOREP_METRIC_X86_ENERGY_PLUGIN`
 
-    The time in usecs, between two consecutive reads of the power/energy values. A longer interval
-    means less disturbance, a shorter interval is more exact.
+    Comma-separated list of sensors, and physical quantity. Can also contain wildcards, e.g.
+    `BLADE/E,*/E`
 
-    On Intel CPUs, the registers are updated roughly every msec. If you choose an interval of 1ms
-    you might find highly variating power consumptions. To gain most exact values, you should set
-    the amplitude to 10, if you can live with less accuracy, you should set it to 100000.
+* `SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN_VERBOSE`/`SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN_VERBOSE` (default `WARN`)
 
-* `SCOREP_X86ENERGY_CPU_PIN`/`VT_X86ENERGY_CPU_PIN` (default=no pinning)
+    Controls the output verbosity of the plugin. Possible values are: `FATAL`, `ERROR`, `WARN`,
+    `INFO`, `DEBUG`. If set to any other value, `DEBUG` is used. Case in-sensitive.
 
-    The CPU to pin the register reading thread to. The reading thread uses the `msr` files of core
-    `<X86_ENERGY_PLUGIN_CPU_PIN>` of each package (default: the first core of a socket). You might
-    consider to set it to a free Hyperthread.
+* `SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN_OFFSET` (default `70000.0`)
 
-* `SCOREP_X86ENERGY_BUF_SIZE`/`VT_X86ENERGY_BUF_SIZE` (default=4M)
+    Can be set to any double variable. Defines the offset in mW that is added to 
+    the blade, because with RAPL only the energy consumption of the
+    CPU is measurable. Blade is the sum of all RAPL values excepted the core
+    because there are included in package.
+    The default value of 70000.0 mW was measured on Taurus
```

Comment:
I don't agree. The results will be even more misleading if just the core and dram energy consumption is measured.

We have seen a lot of "energy optimisations" for the core freq, which completely neglects the fact the the blade is consuming energy as well, which leads to totally inefficient optimisations.

Therefore it is better to assume some offset here than no offset, as no offset is simply not true for all systems.